### PR TITLE
nicotine+: update to 3.3.2.

### PR DESCRIPTION
**Pull request by yoshiyoshyosh**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (`x86_64-glibc`)
- Built locally for:
  - `i686-glibc`

### CODE REVIEW

File: srcpkgs/nicotine+/template [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Template file for 'nicotine+'
 pkgname=nicotine+
-version=3.2.9
-revision=2
+version=3.3.0
+revision=1
 build_style=python3-module
 hostmakedepends="python3-setuptools gettext"
-depends="gspell gtk+3 python3-gobject"
+depends="gspell gtk4 python3-gobject"
```
Comment (mathiascode) [inline srcpkgs/nicotine+/template:7]: gspell is no longer used (only compatible with GTK 3)

Comment (yoshiyoshyosh) [reply srcpkgs/nicotine+/template]: thanks

**Comment by classabbyamp** [comment]
it seems all the icons and translation files moved from `/usr/share` to inside site-packages

**Comment by yoshiyoshyosh** [comment]
> it seems all the icons and translation files moved from `/usr/share` to inside site-packages

is this necessarily a bad thing?

**Comment by classabbyamp** [comment]
if they don't get used by the application, yes

**Comment by mathiascode** [comment]
> it seems all the icons and translation files moved from `/usr/share` to inside site-packages

Private, unthemable icons did indeed move to site-packages, but icons that are used by the system (such as launcher and tray icons) are still located in /usr/share/icons.

Translation files were also moved to simplify i18n-related code for e.g. Flatpak, but if it turns out distros have issues with this, they could be moved back in a future release.

**Comment by cinerea0** [comment]
Superseded by https://github.com/void-linux/void-packages/commit/6f48fbaab4802a5f8452d9333000a3b1336f0b5f; closing.

**Pull request closed without merging**